pages/vi/vi-github-and-markdown.md

```diff
@@ -58,15 +58,17 @@ First, you will need to find the correct GitHub repository to fork. Forking is e
 
 ### Go to Settings and rename your repository
 
-After you fork the repository, you will be on your page: &lt;username&gt;/open-learning-exchange.github.io. Then, you should go to Settings at the top middle of the page and rename your repository to your GitHub username. See example below.
+After you fork the repository, you will be on your page: &lt;username&gt;/open-learning-exchange.github.io. Then, you should go to Settings at the top middle of the page and rename your repository to your GitHub username.  See example below.
 
 ![Renaming Repository](images/vi-rename-repository.png)
 
+Check [FAQ Q16](http://open-learning-exchange.github.io/#!pages/vi/vi-faq.md) if you already have a repository named YourUserName.github.io.
+
 ### Post a link to your repository in chat
 
 After you rename it, make sure to post a link to the github.io website you've just created (&lt;username&gt;.github.io) in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
 
-> @dogi  please look at  https://YourUserName.github.io 
+> @dogi  please look at  https://YourUserName.github.io
```

Comment:
I understand trailing whitespace is not optimal, but these change are not related to the original issue and it makes the PR seem to have more lines of change than it needed. If you are actually work on those lines that have trailing whitespace, it would be fine to remove them.